packages/pages/src/dev/server/server.ts

```diff
@@ -191,9 +191,18 @@ export const createServer = async (
     }
   });
 
+  app.post(
```

Comment:
If you're not passing the override as a param, does app.use encompass app.post too? Basically, do we need this?

Reply:
We need it separate so that it's forced to use `serverRenderRoute` instead of `serverRenderSlugRoute`.